portfolio/src/main/java/com/google/sps/comments/ListCommentsServlet.java

```diff
@@ -81,19 +88,24 @@ private void retrieveComments() {
     //Execute query
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
+    User user = GetUserServlet.getUserData();
     //Applies limit in the number of comments
     List<Entity> resultsLimited = results.asList(FetchOptions.Builder.withLimit(numberOfComments));
 
     for (Entity entity : resultsLimited) {
       //Reads data from an entity
       long commentId = entity.getKey().getId();
       String commentUsername = (String) entity.getProperty("username");
+      String commentUserId = (String) entity.getProperty("userId");
       Date commentTimestamp = (Date) entity.getProperty("timestamp");
       String commentMessage = (String) entity.getProperty("message");
       long votes = (long) entity.getProperty("votes");
 
+      //check if the comment is posted by the same user logged in
```

Comment:
```suggestion
      // Check if the comment is posted by the same user logged in.
```
Put space and capitalize comments.